<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GMT_psxy</class>
 <widget class="QDialog" name="GMT_psxy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>33</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="J_Layout"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>63</y>
     <width>431</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="R_Layout"/>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;按TAB键可在各选项间快速切换&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="g_color">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>-G 填充色</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="L_option">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>311</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>-L 闭合多边形。本项未勾选时，-G对于画线段无效</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="draw_symbol">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>87</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>画符号</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="draw_line">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>画线段或多边形</string>
   </property>
  </widget>
  <widget class="QComboBox" name="S_option">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>-St △三角形</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-Sa 🟉五角星</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-Sc 圆</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-Sx 叉号</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-Sd ⬦菱形</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-Sg ⯃八边形</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-Sh ⬣六边形</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-Si ▽倒三角</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-Sn ⯂五边形</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-Ss □正方形</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-S- 短横线</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-S+ 加号</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-Sy 短竖线</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="W_pen_color">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>线颜色</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>79</x>
     <y>91</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-W</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="W_pen_w">
   <property name="geometry">
    <rect>
     <x>97</x>
     <y>90</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>参考：0.5p</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>216</x>
     <y>91</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#b5b5b5;&quot;&gt;-W 符号轮廓或线段宽度以及颜色&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_datafile">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>选择数据文件</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="datafile">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>仅支持多行文本文件，每行 x y 为每个点坐标</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="other_options">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>209</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#868686;&quot;&gt;剩余未涉及选项可自由添加：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bok">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>250</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bexit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>250</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="size">
   <property name="geometry">
    <rect>
     <x>172</x>
     <y>120</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>参考: 0.1i</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>236</x>
     <y>122</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#b5b5b5;&quot;&gt;-S 符号大小(size)用外接圆直径表示&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>g_color</tabstop>
  <tabstop>W_pen_w</tabstop>
  <tabstop>W_pen_color</tabstop>
  <tabstop>draw_symbol</tabstop>
  <tabstop>S_option</tabstop>
  <tabstop>size</tabstop>
  <tabstop>draw_line</tabstop>
  <tabstop>L_option</tabstop>
  <tabstop>b_datafile</tabstop>
  <tabstop>datafile</tabstop>
  <tabstop>other_options</tabstop>
  <tabstop>bok</tabstop>
  <tabstop>bexit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
